mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -127,6 +135,10 @@ CLASS none/sg net/minecraft/entity/Entity
 	METHOD a damage (Lnone/rr;F)Z
 		ARG 0 source
 		ARG 1 damage
+	METHOD a (Lnone/sg;)V
```

Comment:
This is odd... this method doesn't have any arguments defined but it does in the other classes here where it's overridden..? I was going to say "don't rename parameters of still unnamed functions".